content/blog/20250214_fluent-package-v5.0.6-has-been-released.md

```diff
@@ -0,0 +1,67 @@
+# fluent-package v5.0.6 has been released
+
+Hi users!
+
+We have released fluent-package [v5.0.6](https://github.com/fluent/fluent-package-builder/releases/tag/v5.0.6).
+fluent-package is a stable distribution package of Fluentd. (successor of td-agent v4)
+
+This is a maintenance release of v5.0.x LTS series.
+As bundled Fluentd was updated to 1.16.7.
```

Comment:
> As bundled Fluentd was updated to 1.16.7.

Is this `As` a conjunction?

Reply:
Just a garbage which was forgot to remove.